lib/vistas/paqueteForm.dart

```diff
@@ -99,7 +83,8 @@ class _PaqueteFormState extends State<PaqueteForm> {
                       keyboardType: TextInputType.number,
                       autofocus: false,
                       style: TextStyle(color: TransportifyColors.primarySwatch),
-                      decoration: TransportifyMethods.returnTextFormDecoration("Peso"),
+                      decoration:
+                          TransportifyMethods.returnTextFormDecoration("Peso"),
```

Comment:
```suggestion
                          TransportifyMethods.returnTextFormDecoration("Peso(kg)"),
```
Es uno de los 'bugs' que encontró Patricio, así nos lo quitamos de encima :)

Reply:
Este bug forma parte de la especificación de la UT de mejora 2751